src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf

```diff
@@ -44,59 +44,59 @@
             </trans-unit>
             <trans-unit id="11">
                 <source>This value is not a valid date.</source>
-                <target>Vrednost nije validan datum.</target>
+                <target>Vrednost nije validna kao datum.</target>
             </trans-unit>
             <trans-unit id="12">
                 <source>This value is not a valid datetime.</source>
-                <target>Vrednost nije validno vreme.</target>
+                <target>Vrednost nije validna kao datum i vreme.</target>
             </trans-unit>
             <trans-unit id="13">
                 <source>This value is not a valid email address.</source>
                 <target>Vrednost nije validna adresa elektronske pošte.</target>
             </trans-unit>
             <trans-unit id="14">
                 <source>The file could not be found.</source>
-                <target>Datoteka ne može biti pronađena.</target>
+                <target>Fajl ne može biti pronađen.</target>
```

Comment:
OK, `fajl` is definitely more recognisable to the broad audience than `datotetka`. 👍